common/src/main/java/com/microsoft/identity/common/internal/request/AdalBrokerRequestAdapter.java

```diff
@@ -28,17 +28,19 @@
 import android.content.Context;
 import android.content.Intent;
 import android.os.Bundle;
-import androidx.annotation.NonNull;
-import androidx.annotation.Nullable;
 import android.text.TextUtils;
 import android.util.Pair;
 
+import androidx.annotation.NonNull;
```

Comment:
I think in the release branch they haven't added support for Androidx as a result of the Intune MAM team issue with AndroidX

Reply:
We have another branch for that. (release/1.0.0-NoAndroidX)